site/_guides/cert-manager.md

```diff
@@ -363,27 +368,36 @@ spec:
 The `kubernetes.io/tls-acme: "true"` annotation tells cert-manager to use the `letsencrypt-staging` cluster-wide issuer we created earlier to request a certificate from Let's Encrypt's staging servers.
 
 The certificate is issued in the name of the hosts listed in the `tls:` section, `httpbin.davecheney.com` and stored in the secret `httpbin`.
-Behind the scenes, cert-manager creates a certificate CRD to manage the lifecycle of the certificate.
+Behind the scenes, cert-manager creates a certificate CRD to manage the lifecycle of the certificate, and then a series of other CRDs to handle the challenge process.
+The process goes `certificate` -> `certificaterequest` -> `order` -> `challenge`. If something goes wrong with this process, you will need to `kubectl get` then `kubectl describe` each of these objects in order to get the full picture.
```

Comment:
This background is useful, but incomplete. How would you feel about dropping this editorial?

Reply:
I'm okay with it. I'll remove.